app/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
+  <head>
+    <title>Angular Intro</title>
+  </head>
+  <body>
+    <main>
+      <section class="container" ng-controller="CowsayController">
```

Comment:
The main problem you are having here is that you are defining your controller twice. You can only have one controller with one name(You have two on lines 8 and 21). All of your functionality should all be nested within one controller. Once you fix that the rest will come easier.